lib/server.js

```diff
@@ -651,12 +654,15 @@ Server.prototype.rm = function rm(route) {
  * You can pass in any combination of functions or array of functions.
  * @public
  * @function use
+ * @param {...*} handlers - A variable number of handler functions * and/or a
```

Comment:
Its been a bit since I wrote this, but I believe I'm using the wildcard implying any type is valid. I don't believe there is a way of specifying multiple types beyond this. I would like to be able to remove the nested array of handlers logic but haven't heard back from @mcavage as to why this was included or if it needs to stick around.

The `...` I believe implies a variadic function.